index.js

```diff
@@ -46,7 +46,24 @@ function Product(product){
         </div>
     `;
 }
-
+function Form(){
+    return `
+        <form>
+            <label for ="title">Title</label> 
+            <input type="text" name="title">
+            <label for ="price">Price</label>
+           <input type="text" name="price">
+            <label for ="creator">Creator</label> 
+            <input type="text" name="creators">
+            <label for ="image">Image</label> 
+            <input type="text" name="image">
+            <label for ="selling_points">Selling_Points</label> 
+            <input type="text" name="Selling_Points">
+            <label for ="id">Id</label> 
```

Comment:
instead of `id` (which is an internal implementation detail), lets use `type`, which could be either `book` or `album`.

Remember that if we're choosing between two options, we also probably want to use a `radio` button (i.e. `type="radio"`) 😉